src/main/groovy/com/blackducksoftware/integration/hub/detect/bomtool/NugetBomTool.groovy

```diff
@@ -65,24 +68,21 @@ class NugetBomTool extends BomTool {
             if (!nugetExecutable) {
                 logger.warn("Could not find a ${executableManager.getExecutableName(ExecutableType.NUGET)} executable")
             }
+            outputDirectory = new File(detectConfiguration.outputDirectory, 'nuget')
         }
 
         nugetExecutable && (containsSolutionFile || containsProjectFile)
     }
 
     @Override
     List<DetectCodeLocation> extractDetectCodeLocations() {
-        def outputDirectory = new File(detectConfiguration.outputDirectory, 'nuget')
-        def sourceDirectory = new File(sourcePath)
-        String inspectorExePath = installInspector(sourceDirectory, outputDirectory, nugetExecutable)
-
-        if (!inspectorExePath) {
+        if (!nugetInspectorExecutable) {
             return []
```

Comment:
Should this return no code locations? I think it should throw an exception. Or installInspector does and this doesn't do a check. I mean it is an exception right? We can't continue and detect will fail (if no code locations found), right?